### ISSUE TITLE
Warning on attribut in media query

### ISSUE THREAD

**Issue by mbaudry** [SimenB/stylint#413]
I got a warning when I lint my code for an attribute declaration, when it's in a media query:

```
 @media $large-2 {
    [data-region].slide-in-rtl {
        animation-duration: 0.5s;
    }

    [data-region].slide-out-ltr {
        animation-duration: 0.4s;
    }
}
```

But this does not trigger a warning (outside the media query):

```
[data-region].slide-out-ltr {
    animation-duration: 0.25s;
}
```

Do you have any idea ??

Thanks !
